Neos.Flow/Classes/Persistence/Doctrine/EntityManagerConfiguration.php

```diff
@@ -242,4 +256,10 @@ public function enhanceEntityManager(Configuration $config, EntityManager $entit
             }
         }
     }
+
+    private function getPsrCacheItemPool(string $cacheIdentifier): CacheItemPoolInterface
```

Comment:
This name sounds like it get's "a normale PSR cache pool", but in fact it returns our custom cache pool, no? So maybe find a better name…

What about `getSecurityContextHashAwareCacheItemPool`?

That being said, why is there `Item` in the name?

Reply:
Because it's a `CacheItemPoolInterface`

Reply:
Well… then why on earth is the `CachePool` not a `CacheItemPool`? 🙄